tracer/src/Datadog.Trace/RemoteConfigurationManagement/IRemoteConfigurationManager.cs

```diff
@@ -17,5 +17,7 @@ internal interface IRemoteConfigurationManager
         void RegisterProduct(Product product);
 
         void UnregisterProduct(string productName);
+
+        void SetCapablity(int index, bool available);
```

Comment:
typo: `SetCapability`

Reply:
Thanks, made me realize I'd forgot to reinstall the spell checker extension :D

Reply:
fixed in 947da4ae1